docs/cli-help.txt

```diff
@@ -13,27 +13,27 @@ cert. Major SUBCOMMANDS are:
   register             Perform tasks related to registering with the CA
   rollback             Rollback server configuration changes made during install
   config_changes       Show changes made to server config during installation
+  update_symlinks      Update cert symlinks based on renewal config file
   plugins              Display information about installed plugins
+  certificates         Display information about certs configured with Certbot
```

Comment:
Does this get generated from `cli.py`? Where did `rename` go? Is `docs` the folder that we don't update until release?